generic/tests/netperf.py

```diff
@@ -138,6 +141,22 @@ def mtu_set(mtu):
         for iface in target_ifaces:
             utils.run(host_mtu_cmd % (iface, mtu), ignore_status=False)
 
+    def tweak_tuned_profile():
+        """
+
+        Tweak configuration with truned profile
+
+        """
+        host_tuned_profile = params.get("host_tuned_profile")
+        guest_tuned_profile = params.get("guest_tuned_profile")
+        error_context.context("Changing the MTU of guest", logging.info)
```

Comment:
"Changing the MTU of guest"
The description doesn't match with code.

Reply:
update done, please help review. Thanks,